andela/bp-esa-frontend
# 164237041 Add a reusable navbar

#### What does this PR do?
This PR adds a reusable nav bar for enabling a user to view navigate to card view and list view reports, it also displays the report filtering user interface.

#### Description of Task to be completed?
- Add a ReportNavBar component
- Add CSS style for the component
- Write the test for the component

#### How should this be manually tested?
1. Clone the repo `https://github.com/andela/bp-esa-frontend`
2. Check out to `bp-esa-frontend` folder and run this command on your terminal `git fetch && git checkout `bg-fix-loading-response-164210111` and run `yarn install` on your terminal
3. Start the application with the command `yarn start`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#164237041](https://www.pivotaltracker.com/story/show/164210111)

#### Postman Documentation
N/A

#### Screenshots (If applicable)
<img width="1675" alt="Screenshot 2019-03-14 at 15 03 48" src="https://user-images.githubusercontent.com/24792513/54355550-6797f380-466a-11e9-885e-69956413d373.png">
